bin/.gitignore

```diff
@@ -0,0 +1,28 @@
+# Compiled class file
```

Comment:
I think you pushed some stuff you shouldn't have? This file shouldn't be here I don't think

Anything inside of `bin/`

Reply:
My bad, removed those